content/courses/program-security/duplicate-mutable-accounts.md

```diff
@@ -364,36 +368,918 @@ which we expect to fail.
 
 ```typescript
 describe("duplicate-mutable-accounts", () => {
-    ...
-    it("Invokes secure instruction", async () => {
-        await program.methods
+	...
+    it("Invoke secure instruction with different players should be successful", async () => {
+    await program.methods
+      .rockPaperScissorsShootSecure({ rock: {} }, { scissors: {} })
+      .accounts({
+        playerOne: playerOne.publicKey,
+        playerTwo: playerTwo.publicKey,
+      })
+      .rpc();
+
+    const p1 = await program.account.playerState.fetch(playerOne.publicKey);
+    const p2 = await program.account.playerState.fetch(playerTwo.publicKey);
```

Comment:
`player` and `player` per 'use full names' https://github.com/solana-foundation/developer-content/blob/main/CONTRIBUTING.md#code